models/src/commonMain/kotlin/com/revenuecat/purchases/kmp/Offerings.kt

```diff
@@ -0,0 +1,30 @@
+package com.revenuecat.purchases.kmp
+
+/**
+ * This class contains all the offerings configured in RevenueCat dashboard.
+ * For more info see https://docs.revenuecat.com/docs/entitlements
+ */
+public data class Offerings(
```

Comment:
same comment here about it being a data class

Reply:
Also, I find it a bit confusing Offerings is a data class and Offering an interface. I think it makes up for a very confusing API tbh, should we aim for consistency and make everything an interface? Or at least related classes. Happy to talk about it

Reply:
I'm ok removing data classes and using only normal classes (It's actually recommended for APIs to avoid breaking changes).

As for interface vs class... My only concern there is increasing the complexity of the project, having to define interfaces for everything and then separate classes implementing them... I'm not opposed however, might be cleaner yeah

Reply:
From talking in Slack, I've removed most data classes, but I've left them as classes for now. Happy to talk more though!

Reply:
I converted all `data` classes to regular classes here: https://github.com/RevenueCat/purchases-kmp/pull/163. 

Moving to `interface`s can be done in a separate PR imo? Let's discuss on Slack whether we should do it. Currently I'm thinking maybe we should.